docs/api/ShallowWrapper/render.md

```diff
@@ -28,7 +28,7 @@ function Bar() {
 ```
 
 ```jsx
-const wrapper = shallow(<Bar />);
```

Comment:
This isn't a typo; this is a ShallowWrapper doc, using the `.render` method - not the top-level `render` API.

Reply:
Excuse my mistake :)